### ISSUE TITLE
Unescaped HTML tags in the title, description and probably other fields in a card or board

### ISSUE THREAD

**Issue by fbroen** [wekan/wekan#5219]
When creating maps, HTML tags are not escaped in various places.

Tested in Version 7.12.0

**Comment by xet7** [comment]
@fbroen 

What maps?

What HTML tags?

What various places?

**Comment by xet7** [comment]
If this is about any new security issue, please follow security disclosure:

https://github.com/wekan/wekan/blob/main/SECURITY.md

**Comment by xet7** [comment]
But not escaping something is OK, when it does not create any kind of alert javascript popup like in XSS, because WeKan uses dompurify:

https://wekan.github.io/hall-of-fame/

**Comment by xet7** [comment]
It is a feature in WeKan, that is is possible to use some markdown and html for formatting.

**Comment by fbroen** [comment]
@xet7
I would like to briefly explain the problem that I discovered today in more detail: We use a board to implement XML export files. For this purpose we also create cards where we mention XML tags.

We noticed the problem when we wrote the following for the title of a card: "The XML tag <person> needs to be changed to <pers>"

After saving the map or title, it only said "The XML tag needs to be changed to".

If html-tags are a feature for formating, is it possible to disable it for all boards or for one board?
